Mathlib/CategoryTheory/Limits/Shapes/Pullback/PullbackCone.lean

```diff
@@ -168,6 +166,12 @@ def ext {s t : PullbackCone f g} (i : s.pt ≅ t.pt) (w₁ : s.fst = i.hom ≫ t
   WalkingCospan.ext i w₁ w₂
 #align category_theory.limits.pullback_cone.ext CategoryTheory.Limits.PullbackCone.ext
 
+/-- The natural isomorphism between a pullback cone and the corresponding pullback cone
+reconstructed using `PullbackCone.mk`. -/
+@[simps!]
+def isoMkSelf (t : PullbackCone f g) : mk t.fst t.snd t.condition ≅ t :=
+  PullbackCone.ext (Iso.refl _) (by simp) (by simp)
```

Comment:
Can you leave out the `(by simp) (by simp)` and let automation take care of it? I haven't tested this.

Similarly for pushouts.

Reply:
There is no automation available here (I think?). Do you think it would be a good idea to add automation like `(... := by aesop_cat)` in the definition of `PullbackCone.ext`?

Reply:
Yeah you could try that, if that's what's done for Cones.ext?

Reply:
Yes it is! Fixing it :)